### PR TITLE
Add website title

### LINKED ISSUE #45: Add H1 on page
Let's add an H1 a bit descriptive of what people or watching in the homepage

### PR DESCRIPTION
**Pull request by goulinkh**
## Done

- Add a title
- Fix the header & footer width to match the main section

## QA

- Go to https://specs-canonical-com-59.demos.haus/
- Make sure that the title is changed
- Make sure that the width of the header and footer looks aligned with the main section

## Issues

Fixes #45

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -98,8 +98,14 @@ main {
   }
 }
 
-.p-navigation__row {
-  max-width: 100%;
+// Match the same width as the fluid-breakout
+.row,
+.u-fixed-width {
+  max-width: calc(2 * 14rem + $grid-max-width) !important;
```
Comment (anthonydillon) [inline src/styles.scss:105]: You can set this via a Vanilla setting:
```scss
$grid-max-width: 100rem;
```

File: templates/_base.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>{% block title %}Canonical specs{% endblock %}</title>
+    <title>{% block title %}Canonical Specifications{% endblock %}</title>
```
Comment (anthonydillon) [inline templates/_base.html]: Nitpick would be that this is not a product name so should be sentence case but not a big deal

File: templates/_base.html [hunk 3]
```diff
@@ -17,19 +17,14 @@
...
-              <img
-                class="p-navigation__image"
-                src="https://assets.ubuntu.com/v1/2fcd45d2-logo-canonical-white.png"
-                alt="Canonical"
-                width="133"
-                height="19"
-                style="min-height: auto"
-              />
-            </a>
+      <div class="row p-navigation__wrapper">
```
Comment (anthonydillon) [inline templates/_base.html]: Why the departure from Vanilla: https://vanillaframework.io/docs/patterns/navigation

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few suggestions in the code.

File: src/styles.scss (hunk 1 above)
Comment (goulinkh) [reply src/styles.scss:105]: As `$grid-max-width` is used by `l-fluid-breakout` as well, changing the variable value will change the `max-width` of the main section as well.
So the only way to match the same width is by using the same formula as the `fluid-breakout` layout.

File: templates/_base.html (hunk 3 above)
Comment (goulinkh) [reply templates/_base.html]: Thanks for the comment, I double checked the docs and found my use case there, as it's not an image asset, the styling was broken but found a similar use case in the docs :+1:

File: src/styles.scss (hunk 1 above)
Comment (anthonydillon) [reply src/styles.scss:105]: I see, thanks for trying. Can you file an issue on Vanilla for that.

**Review by anthonydillon — APPROVED** [review]
Looks a little off on small screens but something we can tidy up later :+1:

File: src/styles.scss (hunk 1 above)
Comment (goulinkh) [reply src/styles.scss:105]: Sure!

**Pull request merged**